lms/templates/courseware/progress.html

```diff
@@ -103,26 +103,27 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
       %endif
 
-      %if credit_course is not None:
+      % if is_credit_course:
         <section class="credit-eligibility">
             <div class="credit-eligibility-container">
                 <div class="eligibility-heading">
                     <h2>${_("Requirements for Course Credit")}</h2>
                 </div>
-                %if credit_course['eligibility_status'] == 'not_eligible':
-                    <span class="eligibility_msg">${student.username}, ${_("You are no longer eligible for this course.")}</span>
-                %elif credit_course['eligibility_status'] == 'eligible':
-                    <span class="eligibility_msg">${student.username}, ${_("You have met the requirements for credit in this course.")}
-                        <a href="#">${_("Go to your dashboard")}</a> ${_("to purchase course credit.")}
+                %if credit_course_requirements['eligibility_status'] == 'not_eligible':
+                    <span class="eligibility_msg">${student.get_full_name()}, ${_("You are no longer eligible for this course.")}</span>
+                %elif credit_course_requirements['eligibility_status'] == 'eligible':
+                    <span class="eligibility_msg">${student.get_full_name()}, ${_("You have met the requirements for credit in this course.")}
+                        ${_("{link} to purchase course credit.").format(link="<a href={url}>{url_name}</a>".format(url = reverse('dashboard'), url_name = _('Go to your dashboard')))}
```

Comment:
This is very wrong from an i18n perspective. Please review http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html